### ISSUE TITLE
Felesleges tartományszűrők megjelenítése

### ISSUE THREAD

**Issue by urbalazs** [vmiklos/osm-gimmisn#3403]
Adottak ezek a [referenciaértékek](https://osm.vasony.hu/missing-housenumbers/budapest_12/view-query) és [szűrők](https://github.com/vmiklos/osm-gimmisn/blob/master/data/relation-budapest_12.yaml) a XII. kerületben:

```
Ámor utca	1
Ámor utca	1/A
Ámor utca	10
Ámor utca	2
Ámor utca	3
Ámor utca	4
Ámor utca	5
Ámor utca	6
Ámor utca	7
Ámor utca	8
Ámor utca	9
```

```
  Ámor utca:
    ranges:
      - {start: '1', end: '9'}
      - {start: '2', end: '10'}
```

Látható, hogy a szűrő teljesen felesleges, nincsenek olyan értékek a referenciában, amelyek kilógnának a megadott tartományból.

Az tűnt fel, hogy a XII. kerületi szűrők közt nagyon sok tartományszűrő van, és ahogy ellenőriztem néhányat, teljesen feleslegesek. Gondolom, hogy ezek még az alkalmazás korai időszakából valók. Ahogy néztem, ez a fajta probléma csak tartományoknál jelentkezik.

Azt javaslom, hogy legyen egy új ellenőrzés:

- Mely házszámok hiányoznak az aktuális szűrők használatával?
- Mely házszámok hiányoznak, ha az aktuális szűrőkből eltávolítjuk a `ranges` tartományokat?

Ha mindkét esetben ugyanazok hiányoznak, akkor a tartományszűrők feleslegesek.

Ennek működnie kellene akkor is, ha vegyesen vannak `ranges` és `invalid` szűrők is. Tehát ha a `ranges` + `invalid` ugyanazokat szűri ki mint az `invalid` önmagában, akkor a tartományszűrők feleslegesek. Ezeket is meg kellene jeleníteni a `.../view-lints` oldalakon.

**Comment by vasony** [comment]
A tartomány szűrők dokumentálták az elméleti min és max értékét  Hibát szerintem nem okoz használatuk.
Csak elmélkedek, hogy van-e érteleme ezeket valahogy megmenteni.

**Comment by urbalazs** [comment]
Ha arra lettek használva, hogy az elméleti min és max értéket tárolják, akkor nem. Sokkal beszédesebb `invalid` szűrőkkel kiszűrni a valóban hibás számokat. Továbbá az issueban lévő szűrő esetén nem is értesülünk arról, ha valamiért megváltozna az elméleti min (például lebontják és parkosítják az utca elejét) vagy a max (például továbbépítik az utcát).

**Comment by vmiklos** [comment]
> Gondolom, hogy ezek még az alkalmazás korai időszakából valók

Igen, eredetileg csak ranges volt, az invalid később lett. Mostanában én is inkább csak invaliddal szoktam szűrni, pont ahogy írod te is, azért, hogy ha az utca végére felhúznak egy újabb házat, azt lássuk. Viszont a ranges arra tök jó, hogy HRSZ jellegű szemetet szűrjünk vele, ha nincs megadva range, akkor kódból amúgy is van egy alap ilyen szűrő.

Lényeg a lényeg, szerintem annak van értelme, hogy a linter szóljon érte, hogyha egy utca szűrői semmit se szűrnek ki.
